storage-management/storage-management.md

```diff
@@ -0,0 +1,85 @@
+---
+description: This article explains how to manage and free up your storage on SharePoint sites.
+---
+
+# Storage Management
+
+## Storage Management in Microsoft 365
+
+**Storage management in Microsoft 365 is important, as it directly impacts your monthly expenses**. 
+
+**Storage management in Microsoft 365 isn't just a routine task - it's a strategic necessity.** Your monthly expenses are directly linked to the efficiency of your storage management, particularly within SharePoint sites connected to your organization's purchased licenses.
+
+After you run out of storage for your SharePoint sites, depending on your plan, you can use the Office 365 Extra File Storage add-on to buy more storage in 1GB increments.
+
+However, **the best thing you can do to avoid additional costs is to clean up your storage proactively** and use tools that are available to manage and optimize the usage of your storage.
+
+* [Read this article provided by Microsoft to learn more about SharePoint storage limits.](https://learn.microsoft.com/en-us/office365/servicedescriptions/sharepoint-online-service-description/sharepoint-online-limits)
+
+* [To find more information and see if your plan is eligible to buy extra storage, explore the following Microsoft article.](https://learn.microsoft.com/en-us/microsoft-365/commerce/add-storage-space?view=o365-worldwide)
+
+## Get Started with Storage Management
```

Comment:
ovdje bih već u titleu stavio da se radi dalje o storage managementu u Pointu
##Storage Management in Syskit Point